src/figureprinter/FigurePainter.java

```diff
@@ -92,7 +146,24 @@ public static void drawRhombus(int length) {
      * @param length specified length of triangle will be drawn
      */
     public static void drawIsoscelesTriangle(int length){
-
+        for (int i = 0; i < length; i++)
+        {
+            for (int j = 1; j <= length - i; j++)
+            {
+                System.out.print("  ");
+            }
+            for (int k = 0; k <= i; k++)
+            {
+                System.out.print("* ");
+            }
+            System.out.println("");
+        }
+        for (int i = 0; i < length-1; i++) {
+            for (int j = 0; j <= i ; j++) {
+                System.out.print("* ");
+            }
+            System.out.println();
+        }
```

Comment:
Էս գրածտ տարբերակը ռան տվեցի ուրիշ պատկեր ստացվեց

Reply:
Էս տարբերակը փորձե

for (int i = 0; i < length; i++) {
            for (int j = 0; j < (length - i - 1); j++) {
                System.out.print("  ");
            }
            for (int j = 0; j < (i * 2 + 1); j++) {
                System.out.print("* ");
            }
            System.out.println();
        }